dags/topline.py

```diff
@@ -0,0 +1,46 @@
+from airflow import DAG
+from datetime import timedelta, datetime
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'amiyaguchi@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2017, 4, 23),
+    'email': ['telemetry-alerts@mozilla.com', 'amiyaguchi@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+
+def topline_dag(dag, mode, instance_count):
+    t0 = EMRSparkOperator(
+        task_id="topline_summary",
+        job_name="Topline Summary View",
+        execution_timeout=timedelta(hours=8),
+        instance_count=instance_count,
+        env={
+            "date": "{{ ds_nodash }}",
+            "bucket": "{{ task.__class__.private_output_bucket }}",
+            "mode": mode
+        },
+        uri=(
+            "https://raw.githubusercontent.com/"
+            "mozilla/telemetry-airflow/master/"
+            "jobs/topline_summary_view.sh"
+        ),
+        dag=dag)
```

Comment:
Nit: Consider returning the resulting task from this function. I know it's not used in the code below, but if later we want to add dependent tasks (ie. the Dashboard output tasks), these will need to be used as upstream dependencies.